src/components/tabs/README.md

```diff
@@ -80,7 +80,7 @@ Find out when to use the tabs component in your service in the [GOV.UK Design Sy
 
       </section>
 
-      <section class="govuk-tabs__panel" id="past-week">
+      <section class="govuk-tabs__panel govuk-tabs__panel--hidden" id="past-week">
```

Comment:
Do we need both the hidden modifier _and_ the selected modifier? Can the tabs not be hidden by default unless they have the `--selected` modifier?

Only having one modifier simplifies the code a little bit and also avoids the possibility of a weird 'undefined' state where a tab has neither the selected modifier nor the hidden modifier.

Reply:
Possibly… for context ,I know that I ended up [refactoring the mobile nav in the Design System](https://github.com/alphagov/govuk-design-system/pull/483/files) so that it didn't have both `active` and `hidden` modifiers, because it was confusing (and broken, because sometimes a menu item had neither modifier, and that scenario was poorly defined – see the first commit) so a little wary of this pattern.

Reply:
I agree it seems there could be a less error prone way of doing this, but given this is consistent with another component and making tab panels hidden by default would be a breaking change, could we raise that as a separate issue?

Reply:
👍